server/controllers/finance/profitCenter.js

```diff
@@ -102,8 +106,43 @@ function lookupProfitCenter (id, codes) {
     });
 }
 
+function getProfitValue (req, res, next){
```

Comment:
This function could also use comments to explain what it does and why it exists.